config/stage.json

```diff
@@ -46,6 +46,7 @@
 		"inline-help": true,
 		"jetpack/agency-dashboard": true,
 		"jetpack/api-cache": true,
+		"jetpack/app-branding": false,
```

Comment:
Feel free to omit the flag in environment configurations when it will be disabled. Feature flags are disabled by default, so if we omit it, it works the same way as if it's specifically disabled.

Reply:
Thanks for the heads up! I've gone ahead to remove the flag from environments it's not used in https://github.com/Automattic/wp-calypso/pull/66814/commits/010dd41ab73769c62485b799cab600ce61e949f1. 🙇‍♀️